e2e/account_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"context"
+	"testing"
+
+	pb "github.com/mesg-foundation/engine/protobuf/api"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/metadata"
+)
+
+func testAccount(t *testing.T) {
+	var address string
+	t.Run("create", func(t *testing.T) {
+		resp, err := client.AccountClient.Create(context.Background(), &pb.CreateAccountRequest{
+			Name:     "user",
+			Password: "pass",
+		})
+		require.NoError(t, err)
+		address = resp.Address
+	})
+
+	t.Run("get", func(t *testing.T) {
+		acc, err := client.AccountClient.Get(context.Background(), &pb.GetAccountRequest{Name: "user"})
+		require.NoError(t, err)
+
+		require.Equal(t, "user", acc.Name)
+		require.Equal(t, address, acc.Address)
+	})
+
+	t.Run("list", func(t *testing.T) {
+		resp, err := client.AccountClient.List(context.Background(), &pb.ListAccountRequest{})
+		require.NoError(t, err)
+
+		require.Equal(t, len(resp.Accounts), 2)
```

Comment:
We should check the content of the Accounts array. Check at least if names match.